lib/mpl_toolkits/axisartist/floating_axes.py

```diff
@@ -26,7 +34,8 @@ def __init__(self, grid_helper, side, nth_coord_ticks=None):
          nth_coord = 0 ->  x axis, nth_coord = 1 -> y axis
         """
 
-        value, nth_coord = grid_helper.get_data_boundary(side)  # return v= 0 , nth=1, extremes of the other coordinate.
+        # return v=0, nth=1, extremes of the other coordinate.
```

Comment:
I don't understand this comment.

Reply:
It's just copy-pasted from what was there previously, but made to fit in 79 characters.
I have a doc PR slowing documenting my understanding of axisartist coming up...

Reply:
I would just remove this comment. It's more confusing than helpful, in particular if you put it on a separate line.

The first half is also the docstring of `get_data_boundary()`. It's better if people check that function instead of trying to figure out what the comment means.

Reply:
fair enough, removed.